toolkit.nu

```diff
@@ -9,9 +9,18 @@
 # check standard code formatting and apply the changes
 export def fmt [
     --check: bool  # do not apply the format changes, only check the syntax
+    --verbose: bool # print extra information about the command's progress
```

Comment:
@amtoine I decided to keep the `--verbose` functionality for enabling the extra `"running"` messages, but the error reporting remains independent from this parameter.

Reply:
i've tested this and it looks perfect to me :blush:

Reply:
are you crazy?? :scream: 

working with you is very pleasing :open_mouth: :heart: